src/components/InputField/InputFieldStyled.js

```diff
@@ -1,6 +1,7 @@
 import styled from 'styled-components';
 
-const getNotificationColor = ({ $isErrorColor, $isSuccessColor }) => {
+const getNotificationColor = ({ $isErrorColor, $isSuccessColor }) =>
+{
   if ($isErrorColor) return 'var(--color-error)';
   if ($isSuccessColor) return 'var(--color-success)';
   return '#79747E';
```

Comment:
I propose to add colors such as '#79747E' to the GlobalStyles

Reply:
Changed all text colors to var(--color-grey-6)